### DISCUSSION TITLE
Move apart the "Delete" and "Restore" options for files in Trash

### DISCUSSION THREAD

**Discussion by ghost** [linuxmint/discussions#313] [Ideas]
In Nemo's Trash, when right-clicking on a file, you see the following context menu:

![image](https://github.com/linuxmint/discussions/assets/2385329/4326a765-e59e-4dbe-824c-2120b4646ea9)

If you want to restore a file, it seems very easy to accidentally click on Delete Permanently instead, since it's right above Restore. I get nervous every time I want to restore a file, fearing I'll accidentally delete it instead.

I recommend moving the menu items apart to avoid such issues.

**Comment by alexeymuranov** [comment]
I would say that moving apart items in a menu so that the user does not accidentally click a wrong one is not a proper way to address the issue of user's clicking in a wrong place. A proper solution would be an option that would activate a confirmation dialogue for the risky action.

**Reply by 100savage** [reply]
Linux Mint already has the confirmation dialog action for the "Delete Permanently" It doesn't have one for the "restore" but I do not think it is needed for the restore action.

**Reply by alexeymuranov** [reply]
@100savage , indeed, thanks. Then this is a non-issue. IMO it can be closed.

**Comment by ghost** [comment]
> Linux Mint already has the confirmation dialog action for the "Delete Permanently"

Ah, so it does. My bad, sorry. Yess, will close this non-issue then.
